### PR TITLE
17 szkielet aplikacji web w oparciu o maven

### PR DESCRIPTION
**Pull request by miosz**

### CODE REVIEW

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/controller/DishController.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.infoshare.myfitwebapp.controller;
+
+import com.infoshare.myfitwebapp.model.DishData;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@RequestMapping("dishes")
+public class DishController {
+
+    @GetMapping("/")
```
Comment (lyingparachute) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/controller/DishController.java:10]: Is "/" necessary?

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/controller/HelloController.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.infoshare.myfitwebapp.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloController {
+
+    @GetMapping("/")
```
Comment (lyingparachute) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/controller/HelloController.java:9]: The same thing here

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/controller/ProductController.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.List;
+
+@RequestMapping("products")
+@Component
+public class ProductController {
+
+    @GetMapping("/productsList")
```
Comment (lyingparachute) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/controller/ProductController.java:14]: Maybe change endpoint to "list" not "/productsList"

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/DishData.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package com.infoshare.myfitwebapp.model;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
```
Comment (AdSten) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/DishData.java:4]: redundant

File: myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/ProductData.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
+package com.infoshare.myfitwebapp.model;
+
+import org.springframework.stereotype.Component;
```
Comment (AdSten) [inline myFit-webApp/src/main/java/com/infoshare/myfitwebapp/model/ProductData.java:3]: redundant

**Review by AdSten — APPROVED** [review]

**Review by lyingparachute — COMMENTED** [review]
Read comments

**Review by MateuszLesisz — APPROVED** [review]

**Pull request merged**